docs/release_notes/release-3.0.md

```diff
@@ -1,5 +1,31 @@
 # StarRocks version 3.0
 
+## 3.0.4
+
+Release date: July 18, 2023
+
+### New Feature
+
+Queries can be rewritten even when the queries contain a different type of join compared with the materialized view of the base tables you want to query. [#25099](https://github.com/StarRocks/starrocks/pull/25099)
```

Comment:
Queries can be rewritten even when the queries contain a different type of join than the materialized view.